session/bootstrap.go

```diff
@@ -357,6 +357,14 @@ const (
 		last_analyzed_at TIMESTAMP,
 		PRIMARY KEY (table_id, column_id) CLUSTERED
 	);`
+	// CreateTableCacheMetaTable stores the cached table meta lock information.
+	CreateTableCacheMetaTable = `CREATE TABLE IF NOT EXISTS mysql.table_cache_meta (
```

Comment:
Could we remove `CreateTableCacheMetaTable`?

Reply:
This may affect the next cherry-pick classmates.